app/assets/stylesheets/components/_answerbox.scss

```diff
@@ -61,6 +61,20 @@
       }
     }
   }
+
+  &__actions {
+    text-align: right;
+    @include media-breakpoint-down('md') {
+      justify-content: space-between;
+      padding-top: 10px;
+    }
+  }
+
+  @include media-breakpoint-down('md') {
+    .card-body {
+      padding-bottom: .6rem;
+    }
+  }
```

Comment:
mobile first, no downwards queries please. Use `media-breakpoint-up` instead. And inverse styling (having styles on mobile, reset on other queries if necessary)